parsel/utils.py

```diff
@@ -80,4 +80,9 @@ def extract_regex(regex, text, replace_entities=True):
     strings = flatten(strings)
     if not replace_entities:
         return strings
-    return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
\ No newline at end of file
+    return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
+
+def shorten_selector_data(data, width):
+    """Shortens the preview of extracted data by adding placeholder '...'
+    """
+    return data[:width - 3] + "..." if (len(data) > width - 3) else data
```

Comment:
I think it is not precise, as it may shorten data unnecessarily. For example, if data is of length 38, 39 or 40, and width is set to 40, only first 37 chars are visible, and "..." is added - truncation is unnecessary in this case.

I'd also prefer to use `sep = '...'`, and then `len(sep)` instead of a magic "3" constant.

Would you also mind adding an extra blank line above this function, to follow pep8?

Reply:
I'd also rename it to avoid mentioning Selectors (as it is just a text shortening function), and add a few tests, likely doctests.